packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -765,14 +765,33 @@ class PageTransitionsTheme with Diagnosticable {
     Animation<double> secondaryAnimation,
     Widget child,
   ) {
-    TargetPlatform platform = Theme.of(context).platform;
+    final TargetPlatform platform = Theme.of(context).platform;
+
+    bool cupertinoTransitionInProgress = false;
 
     if (CupertinoRouteTransitionMixin.isPopGestureInProgress(route)) {
-      platform = TargetPlatform.iOS;
+      cupertinoTransitionInProgress = true;
+    }
```

Comment:
Could this be simplified to just

```suggestion
final bool cupertinoTransitionInProgress = CupertinoRouteTransitionMixin.isPopGestureInProgress(route);
```